### ISSUE TITLE
Accessibility violation: #search > ul > li > a.dropdown-toggle does not have discernible text

### ISSUE THREAD

**Issue by galvarez421** [PurdueMarketingAndMedia/purdueTemplates-2015#12]
Related to #11, `#search > ul > li > a.dropdown-toggle` element does not have discernible text. This is a violation of WCAG 2.0 A and Section 508, according to https://dequeuniversity.com/rules/axe/2.4/link-name.

Issue detected using the following Chrome extensions:

- aXe (https://dequeuniversity.com/rules/axe/2.4/link-name)
- WAVE Evaluation tool

Recommended fix: Add `aria-label="Search"` to element. This clears the issue for the 2 extensions listed above.

**Comment by jacobvenable** [comment]
On second look, I'm not sure that this is correct. Currently, the content is being filled via a CSS ::after selector. If you use VoiceOver to focus on the button, it does state, "Search". Although, I agree that adding an aria-label attribute or visibly hidden text would be a better fix.
